pom.xml

```diff
@@ -16,6 +16,123 @@
     <version>1.1.3</version>
   </parent>
 
+  <properties>
+    <munit.version>2.3.14</munit.version>
+    <mule.maven.plugin.version>3.8.1</mule.maven.plugin.version>
+    <munit.extensions.maven.plugin.version>1.1.3</munit.extensions.maven.plugin.version>
+    <jacoco.maven.plugin.version>0.8.2</jacoco.maven.plugin.version>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
```

Comment:
I think the main thing needed here - if it exists - is a reference to where you got all this config. If it's just the 1.x connector, then don't reference that. But if there's a Mule docs page, that'd be great to reference here so that if we run into a problem with the config in the future (which happened with the 1.x connector for me), we know where to start looking for help.